docs/frontend_utilisation_guide.md

```diff
@@ -76,15 +120,21 @@ document.body.onload = function () {
 ```typescript
 const config = new Config({ useUrlParams: true, initialSettings: { OfferToReceive: true, TimeoutIfIdle: true } });
 ```
-6) Create the `stream` object and pass in the new `config` object
+6) Create an if statement that will make use of the `webSocketAddress` variable if one is included 
 ```typescript
-const stream = new PixelStreaming(config);
+if(webSocketAddress != ""){
+	config.setTextSetting(TextParameters.SignallingServerUrl, webSocketAddress)
+}
 ```
-7) Create the `spsApplication` object and pass in the `stream` object enclosed in curly braces
+7) Create `stream` and `spsApplication` instances that implement the Epic Games Pixel Streaming Frontend PixelStreaming and Application types
```

Comment:
can you clarify to be clearer about what is implemented:
> that implement the Epic Games Pixel Streaming Frontend PixelStreaming and Application types

Reply:
Resolved in 52bd7bae3ee7504b2807ebd00209bd2ab64e3233